one/src/main.rs

```diff
@@ -81,27 +95,50 @@ struct DaemonOpts {
     #[arg(long, default_value_t = false, env = "CERAMIC_ONE_TRACING")]
     tracing: bool,
     /// Unique key used to find other Ceramic peers via the DHT
-    #[arg(
-        long,
-        default_value = "/ceramic/testnet-clay",
-        env = "CERAMIC_ONE_NETWORK_ID"
-    )]
-    network_id: String,
+    #[arg(long, default_value = "testnet-clay", env = "CERAMIC_ONE_NETWORK")]
+    network: Network,
+
+    /// Unique key used to find other Ceramic peers via the DHT
+    #[arg(long, env = "CERAMIC_ONE_LOCAL_NETWORK_ID")]
+    local_network_id: Option<u32>,
+
+    /// When true mdns will be used to discover peers.
+    #[arg(long, default_value_t = false, env = "CERAMIC_ONE_MDNS")]
+    mdns: bool,
+}
+
+#[derive(ValueEnum, Debug, Clone)]
+enum Network {
```

Comment:
why is this not the enum from core/src/network ?

Reply:
This enum requires that it implement the ValueEnum trait so it can be used as a CLI argument. This decouples CLI parsing from the actual network enum. Specifically the CLI parsing does not allow enum variants that contain values so the `core::Network::Local(u32)` variant is not allowed to implement the ValueEnum trait. So this provides a clean mapping between them.